src/features/PatientInfo/index.js

```diff
@@ -16,39 +16,10 @@ function PatientInfo(props) {
       <h1 className="patient-name">{input.name}</h1>
       <div className="patient-detail">
         <div className="id">
-          {/* <div className="float-end"><a href="fake_ehr.html" className="view">View patient in EHR</a></div> */}
+          <div><b>Date of Birth:</b> <span><time dateTime={dob}>{dob}</time></span> ({input.age} y.o.)</div>
           <div><b>MRN:</b> <span>{mrn}</span></div>
           <div><b>Pregnant:</b> <span>{input.isPregnant === false ? 'No' : input.isPregnant === true ? 'Yes' : null}</span></div>
         </div>
-        <div className="info-items">
-          <Row>
-            <Col>
```

Comment:
`Row` and `Col` imports can now be removed from line 1, since they are no longer used within this file.

Reply:
I addressed this in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/54/commits/1f1a3a370691dc05d209fe5de90617a6988ded10.

Reply:
Good catch.